libs/NfcCoreLib/lib/NciCore/phNciNfc_Core.c

```diff
@@ -614,6 +615,12 @@ phNciNfc_CoreValidateNfceeMgtCmd(pphNciNfc_CoreTxInfo_t pTxInfo,
                 wStatus = NFCSTATUS_SUCCESS;
             }
             break;
+        case phNciNfc_e_NfceeMgtPowerAndLinkCtrlCmdOid:
+            if ((PHNCINFC_CORE_NFCEEMODESET_CMD_PAYLOADLEN == pTxInfo->wLen) && (NULL != pTxInfo->Buff))
```

Comment:
PHNCINFC_CORE_NFCEEPOWERANDLINKCTRL_CMD_PAYLOADLEN

Reply:
This is a mistake and now updated.